src/components/reservation/helper.ts

```diff
@@ -0,0 +1,17 @@
+import { AgencyBranch } from "../../core/fbs/model";
+
+export const getPreferredLocation = (id: string, array: AgencyBranch[]) => {
+  const locationItem = array.find((item) => item.branchId === id);
+  return locationItem ? locationItem.title : id;
+};
+
+export const getPreferredLocationText = (
```

Comment:
Consider renaming location to branch in these function names.

Branch is a concept used by FBS so unless we have a good reason we could just continue using that concept instead of introducing our own.